content/en/docs/instrumentation/python/getting-started.md

```diff
@@ -251,7 +260,147 @@ automatically created one:
 The `parent_id` of `do_roll` is the same is the `span_id` for `/rolldice`,
 indicating a parent-child reletionship!
 
-## Send traces to an OpenTelemetry Collector
+### Metrics
+
+Now modify `app.py` to include code that initializes a meter and uses it to
+create a counter instrument which counts the number of rolls for each possible
+roll value:
+
+```python
+# These are the necessary import declarations
+from opentelemetry import trace
+from opentelemetry import metrics
+
+from random import randint
+from flask import Flask, request
+
+tracer = trace.get_tracer(__name__)
+# Acquire a meter. There's one set up for you globally,
+# and it's also used by opentelemetry-instrument.
```

Comment:
Is this accurate? Did you mean to say provider is set up globally?

Reply:
Just remove the comment or keep `Acquire a {tracer,meter}.` part. I don't think there is any concept of setting up global tracer and use it. I believe the original author wanted to mention about the provider instead, which `opentelemetry-instrument` does in `initialize_components`.

Reply:
Good points, done.